app/src/main/res/values/strings.xml

```diff
@@ -163,6 +164,8 @@
   <string name="message_notification_summary">Message</string>
   <string name="message_notification_text">New incoming message</string>
   <string name="pref_general_about_title">About</string>
+  <string name="free_draw_title">Free draw</string>
```

Comment:
One can be a reference to another:
`menu_action_free_draw_title` & `free_draw_title`
(Unless it is intended for separation)